packages/ubuntu_desktop_installer/lib/l10n/app_oc.arb

```diff
@@ -39,13 +39,30 @@
   "installThirdPartyTitle": "Installar de logicials tèrces pel material grafic e Wi-Fi e dels formats de mèdia suplementaris",
   "installThirdPartySubtitle": "Aqueste logicial es somés a de tèrmes de licéncia incluses dins sa documentacion. Certans son proprietaris.",
 
-  "allocateDiskSpace": "Atribuir l’espaci disc",
-  "startInstallingButtonText": "Començar l’installacion",
+  "allocateDiskSpace": "Atribuir l'espaci disc",
+  "startInstallingButtonText": "Començar l'installacion",
   "diskHeadersDevice": "Disc",
   "diskHeadersType": "Tipe",
   "diskHeadersMountPoint": "Punt de montatge",
   "diskHeadersSize": "Talha",
   "diskHeadersUsed": "Utilizat",
   "diskHeadersSystem": "Sistèma",
-  "diskHeadersFormat": "Formatar"  
-}
+  "diskHeadersFormat": "Formatar",
+  
+  "writeChangesToDisk": "Aplicar las modificacions suls disques",
+  "writeChangesFallbackSerial": "disc",
+  "writeChangesDescription": "Se contunhatz, las modificacions çai-jos seràn escritas suls disques. Poiretz realizar de cambiaments mai tard a la man.",
+  "writeChangesPartitionTablesHeader": "Las taulas de particions dels periferics seguents seràn modificadas :",
+  "writeChangesPartitionTablesEntry": "{serial} ({path})",
+  "writeChangesPartitionsHeader": "Las particions seguentas seràn formatadas :",
+  "writeChangesPartitionEntryPrimaryFull": "particion n° {partitionNumber} de {diskSerial} ({diskPath}) de tipe {fstype} coma {mountPath}",
+  "writeChangesPartitionEntryPrimary": "particion n° {partitionNumber} de {diskSerial} ({diskPath}) coma",
+  "writeChangesPartitionEntrySecondary": "        particion n° de tipe {fstype} coma {mountPath}",
+
+  "chooseYourLookPageTitle": Causissètz l'aparéncia",
```

Comment:
That string is missing an opening double quote.

Reply:
changed!